src/tests/Dashboard/reducer/conceptReducer.test.js

```diff
@@ -281,4 +283,17 @@ describe('Test suite for concepts reducer', () => {
     state = reducer(initialState, { type: REMOVE_SELECTED_SET, payload: setToRemoveKey });
     expect(state.selectedSets).not.toContainEqual(existingSet);
   });
+
+  describe(REPLACE_CONCEPT, () => {
+    it('should replace a concept in dictionaryConcepts and paginatedConcepts', () => {
+      let state = reducer(multipleConceptsMockStore.concepts, 'init');
+      const conceptToReplace = state.dictionaryConcepts[0];
+      const newConcept = concepts;
+      newConcept.id = conceptToReplace.id;
+      expect(conceptToReplace).not.toEqual(newConcept);
+      state = reducer(state, replaceConcept(newConcept));
```

Comment:
How sure can we be that these were not so before the line above?

Reply:
The assertion on line 293 above accounts for this, since `conceptToReplace` is `state.dictionaryConcepts[0]`.(line 290).